src/SFA.DAS.QnA.Application/Queries/Sections/GetSequenceSections/GetSequenceSectionsHandler.cs

```diff
@@ -58,7 +59,7 @@ private static void RemoveInactivePages(Section section)
             section.QnAData.Pages.RemoveAll(p => !p.Active);
         }
 
-        private  void RemovePagesBasedOnNotRequiredConditions(Section section, JObject applicationData)
+        private  void RemovePagesBasedOnNotRequiredConditions(Section section, JsonObject applicationData)
```

Comment:
extra whitespace

Reply:
not visible in visual studio

Reply:
![image](https://user-images.githubusercontent.com/119664159/220090152-84b7d7d2-e38a-4353-89bd-a1d2c5b02119.png)

Reply:
done